bin/setup.js

```diff
@@ -0,0 +1,32 @@
+/* eslint no-console: 0 */
+var fs = require('fs')
+var resolve = require('path').resolve
+var join = require('path').join
+var cp = require('child_process')
+var os = require('os')
+
+var root = resolve(__dirname, '..')
+var docs = resolve(__dirname, '..', 'docs')
+var isWindows = os.platform().startsWith('win')
+var npmCmd = isWindows ? 'npm.cmd' : 'npm'
+var rmrf = isWindows ? 'if exist "node_modules" (rd /s /q node_modules)' : 'rm -rf node_modules'
+
+cp.execSync('cls', { env: process.env, stdio: 'inherit' })
+
+install(root)
+install(docs)
+
+function install(dir) {
+  console.log('working on ' + dir)
+
+  if (!fs.existsSync(join(dir, 'package.json'))) { return }
```

Comment:
Why this return?

Reply:
because i copied from StackOverflow that was originally for scanning folders. This was there originally as check for something else but rather than delete it i thought it was a nice check just incase the wrong folder or empty folder was passed in. 
Although as i write this i realise that it should console log if thats the case.